packages/strapi-plugin-graphql/services/build-aggregation.js

```diff
@@ -252,15 +252,15 @@ const preProcessGroupByData = function({ result, fieldKey, filters }) {
  */
 const createGroupByFieldsResolver = function(model, fields) {
   const resolver = async (filters, options, context, fieldResolver, fieldKey) => {
-    const params = {
+    const params = convertRestQueryParams({
```

Comment:
Can you call this filters to match the other one ?

Reply:
It's already matched. [createAggregationFieldsResolver](https://github.com/olragon/strapi/blob/62975547dd8bc16a1a4f977992ea74c7f13a7df8/packages/strapi-plugin-graphql/services/build-aggregation.js#L179) & [createGroupByFieldsResolver](https://github.com/olragon/strapi/blob/62975547dd8bc16a1a4f977992ea74c7f13a7df8/packages/strapi-plugin-graphql/services/build-aggregation.js#L255)

Reply:
the name of the varialbe is filers and here you are using params :D

Reply:
Sorry for your confusion, I don't familiar with exists code for Mongoose so I keep it intact. Let's fix it :-p

Reply:
Let's review the code with a birds-eye view. `filters` need to pass as is to the subfield so we need temporary `params` or any other name. It's Mongoose code intention.

![image](https://user-images.githubusercontent.com/47636/78029600-1220a880-738b-11ea-9b52-7ff760d379c9.png)

Reply:
Ok let's not change this then.